### ISSUE TITLE
Missing stdlib\genglsl shaders while running MaterialX viewer

### ISSUE THREAD

**Issue by hb3p8** [materialx/MaterialX#453]
Hi! I've tried to run MaterialX viewer from latest (on 30-Jul-2020) prebuilt distribution, but it complaints about stdlib\genglsl shaders.

Output:
```
MaterialX_Windows_VS2017_x64_Python37\bin>MaterialXView.exe
Failed to generate environment shader: Could not find source file 'stdlib\genglsl\mx_image_color3.glsl' used by implementation 'IM_image_color3_genglsl'
Failed to generate wireframe shader: Could not find source file 'stdlib\genglsl\mx_constant.inline' used by implementation 'IM_constant_color3_genglsl'
Failed to generate shadow shader: Could not find source file 'stdlib\genglsl\mx_constant.inline' used by implementation 'IM_constant_color3_genglsl'
Failed to generate shadow blur shader: Could not find source file 'stdlib\genglsl\mx_image_color3.glsl' used by implementation 'IM_image_color3_genglsl'
Caught exception in main loop: Could not find source file 'stdlib\genglsl\mx_constant.inline' used by implementation 'IM_constant_color3_genglsl'
```
I have used the link from repo readme: https://ci.appveyor.com/api/projects/jstone-lucasfilm/materialx/artifacts/build%2FMaterialX_Windows_VS2017_x64_Python37.zip?job=Environment%3A%20APPVEYOR_BUILD_WORKER_IMAGE%3DVisual%20Studio%202017%2C%20GENERATOR%3DVisual%20Studio%2015%202017%2C%20TOOLSET_NAME%3DVS2017%2C%20ARCH%3Dx64%2C%20PYTHON%3DC%3A%5CPython37-x64%2C%20PYTHON_NAME%3DPython37

**Comment by jstone-lucasfilm** [comment]
Thanks for reporting this, Danila, and I'm seeing the same issue in the latest development build.  While we're investigating this, you can use the pre-built Windows binaries found here, which are locked to the v1.37.1 release of MaterialX:

https://github.com/materialx/MaterialX/releases/tag/v1.37.1

**Comment by jstone-lucasfilm** [comment]
This has been fixed in pull request #456.  Thanks for bringing this to our attention, @hb3p8!
